fix typo in a comment

ithinuel opened the pull request:
### Description

This fixes a dirty copy&paste.

### Pull request type

    [x] Fix
    [ ] Refactor
    [ ] New target
    [ ] Feature
    [ ] Breaking change

ithinuel committed:
fix typo in a comment

0xc0170 commented:
/morph build

ithinuel commented:
This failure is unrelated to the changes brought by this PR.

cmonr commented:
/morph test

0xc0170 commented:
/morph test

cmonr commented:
/morph test

ithinuel commented:
@0xc0170 @cmonr Are the tests ran on a "merged" branch or on this specific head ?
So I need to rebase ?

cmonr commented:
The tests are run on the base branch of this PR, with the PR being merged into it. No idea why CI is failing like this. I suspect this still has older tests that experience intermittent issues.

@studavekar Ideas?

0xc0170 commented:
> @0xc0170 @cmonr Are the tests ran on a "merged" branch or on this specific head ?
So I need to rebase ?

I would recommend rebasing (if its older than a week or so) and we restart the test.

ithinuel commented:
rebased on top of the target branch

cmonr commented:
Will run CI once it frees up from release builds.

0xc0170 commented:
/morph build

cmonr commented:
Halting CI builds until RC3 PRs are completed. Will resume after.

0xc0170 commented:
/morph test
/morph export-build

cmonr commented:
Pausing CI until 5.9 RC3 completes CI. Will restart jobs when able.

cmonr commented:
@ithinuel Can you look at the pr-head issues? It appears as if something is being declared implicitly, but imo, this PR should not have caused anything to fail.

FYI, @ARMmbed/mbed-os-maintainers might need to look into why pr-head failed on this a bit more closely.

ithinuel commented:
@cmonr indeed I forgot a flag while rebasing >< I fixed that in feature-hal-spec-sai-rebased on my fork so you can push it as the new PR head.
I updated this PR to be on top of the new PR-head.

cmonr commented:
/morph build
